### ISSUE TITLE
Unregister and Unsubscribe when client disconnects

### ISSUE THREAD

**Issue by hugomallet** [christian-raedel/nightlife-rabbit#5]
The problem:
1. A client registers a procedure
2. Client disconnects unexpectedly
3. Client reconnects
4. Client tries to registers back the procedure
5. The error "wamp.error.procedure_already_exists" is raised

Does the removeSession function in the router should also unregister the registered procedures and unsubscribe to subscribed topics?

Thank you,
Hugo

**Comment by solognt** [comment]
Yes I have the same issue. Seems like they solved it in crossbar.io by adding a auto ping functionality.
      "auto_ping_interval": 1000,
      "auto_ping_timeout": 2000,
      "auto_ping_size": 4

Maybe that is an option for nightlife-rabbit too?

See this discussion:
https://github.com/crossbario/autobahn-js/issues/127

**Comment by hugomallet** [comment]
Thanks @francois06

fixed by #7
